### PR TITLE
Provide name for components

### PR DESCRIPTION
**Pull request by matthijsgroen**
This helps with using the React dev tools to inspect elements
from the browser.

### CODE REVIEW

**Comment by matthijsgroen** [comment]
Sorry for all the `prettier` commit stuff!

File: project_template/src/components/Icon.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
   const iconAsset = require(`src/assets/icons/${icon}.svg`).default;
-  return <svg {...props}><use xlinkHref={iconAsset.url} /></svg>;
+  return (
+    <svg {...props}>
+      <use xlinkHref={iconAsset.url} />
+    </svg>
+  );
 };
+
+export default Icon;
```
Comment (pascalw) [inline project_template/src/components/Icon.js]: Does this really make a difference? With this change do you see the name `Icon` anywhere while debugging?

Comment (matthijsgroen) [reply project_template/src/components/Icon.js]: Yes using the React dev tools, you will see `Icon` instead of `Unknown`

Comment (pascalw) [reply project_template/src/components/Icon.js]: Ah ok, I googled it and indeed this seems the recommended approach.
This apparently also influences the names that show up in stacktraces, so this is quite useful in that regard too.

**Pull request merged**